ansible/pbTestScripts/qemuPlaybookCheck.sh

```diff
@@ -237,11 +237,29 @@ done
      	  $EXTRA_ARGS \
 	  -nographic) > "$workFolder/${OS}.${ARCHITECTURE}.startlog" 2>&1 &
 
-	echo "Machine is booting; logging console to $workFolder/${OS}.${ARCHITECTURE}.startlog Please be patient"
-	sleep 180
-	tail "$workFolder/${OS}.${ARCHITECTURE}.startlog" | sed 's/^/CONSOLE > /g'
-	echo "Machine has started, unless the above log shows otherwise ..."
+	echo "Machine is booting; logging console to $workFolder/${OS}.${ARCHITECTURE}.startlog"
+	echo "Please be patient"
 
+	SECONDS=0
+	while [ true ];
+	do
+		if tail "$workFolder/${OS}.${ARCHITECTURE}.startlog" | grep -q login; then
```

Comment:
Why `tail` instead of a straight grep of the log - does `login` appear earlier on?
It's always possible  (though unlikely!) that another ten lines of stuff will appear in the 10 seconds since the line we want. Would be good to add a comment if there's a good reason for this additional complexity

Reply:
I couldn't say for sure that `login` doesn't appear in the rest of the log - I just used tail as I knew it was definitely at the end. In my experience, additional stuff doesn't extend past 10 lines, however again, I couldn't say for sure. 
I could up it to say 15 or 20 lines using the `-n` argument?